### PR TITLE
How to implement webhooks

### PR DESCRIPTION
**Pull request by multi-Adams**
created the How to Implement Webhooks article

### CODE REVIEW

File: content/blog/how-to-implement-webhooks.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+In this comprehensive guide, we will delve into the concept of Webhooks, their significance in web development and explore the step-by-step process of implementing Webhooks effectively. Whether you're a seasoned developer seeking to expand your toolkit or a newcomer eager to grasp the intricacies of Webhooks, this article will equip you with the knowledge and practical know-how to harness the full potential of this transformative technology.
+
+> If you're new to the world of webhooks, check out our [detailed guide](getconvoy.com/blog/) on what webhooks are and why they are the go-to communication mechanism for major technology companies like GitHub, Stripe, Shopify, and many others.
+
+Let's explore how to implement webhooks in the following sections.
+
+## Implementing Webhooks
+
+Webhooks involve three primary actors: the sender and the receiver. Understanding these is crucial to implementing webhooks effectively.
```
Comment (danvixent) [inline content/blog/how-to-implement-webhooks.md:29]: Hey, you only listed two primary actors

Comment (multi-Adams) [reply content/blog/how-to-implement-webhooks.md:29]: Hi,

I'm so sorry. I missed this comment.

Yes. During the review, @jirevwe mentioned that it should be 2 primary actors, and the rest should be called components as the 2 primary actors are sending them.

See here:
image

Please let me know what you think.

Comment (danvixent) [reply content/blog/how-to-implement-webhooks.md:29]: Hi @multi-Adams, @jirevwe is correct, but i see the article still reads "Webhooks involve three primary actors".

Comment (multi-Adams) [reply content/blog/how-to-implement-webhooks.md]: That's true!

I have fixed it; thanks for pointing that out.

I'll go ahead and resolve this conversation.

**Pull request closed without merging**